tools/validate-deployment.sh

```diff
@@ -60,7 +62,7 @@ if [[ $ipaddress =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
         echo "[ERROR] Public key missing when resetting key for ProctorVM for $teamAAD - Subscription $subid - Exiting ..."
     fi
 
-    scp -o StrictHostKeyChecking=no -i $ID_RSA_PRIVATE -r azureuser@$ipaddress:/home/azureuser/team_env/* ./$teamAAD/
+    scp -o StrictHostKeyChecking=no -i $ID_RSA_PRIVATE -r azureuser@$ipaddress:/home/azureuser/logs/* ./$teamAAD/
```

Comment:
This should get the files from the nginx content directory
`scp -o StrictHostKeyChecking=no -i $ID_RSA_PRIVATE -r azureuser@$ipaddress:/home/nginx/contents/* ./$teamAAD/
`

Reply:
@dcaro so it shouldnt get from `/home/azureuser/logs` but instead from `/nginx/contents` ?

Reply:
No, /home/nginx/contents. the kvstore and kubeconfig files are in that directory.